packages/react-data-grid/src/Row.js

```diff
@@ -155,7 +155,11 @@ const Row = React.createClass({
   },
 
   getExpandableOptions(columnKey) {
-    return { canExpand: this.props.subRowDetails && this.props.subRowDetails.field === columnKey, expanded: this.props.subRowDetails && this.props.subRowDetails.expanded, children: this.props.subRowDetails && this.props.subRowDetails.children, treeDepth: this.props.subRowDetails ? this.props.subRowDetails.treeDepth : 0 };
+    let subRowDetails = this.props.subRowDetails;
+    if (subRowDetails) {
+      return { canExpand: subRowDetails && subRowDetails.field === columnKey && subRowDetails.children && subRowDetails.children.length > 0, field: subRowDetails.field, expanded: subRowDetails && subRowDetails.expanded, children: subRowDetails && subRowDetails.children, treeDepth: subRowDetails ? subRowDetails.treeDepth : 0, subRowDetails: subRowDetails };
+    }
```

Comment:
Need a test that the correct props are passed to a cell when subRowDetails prop is defined